### PR TITLE
[BREAKING] add checkbox field, rename old checkbox and radio

### LINKED ISSUE #1: Confirmation checkbox field
It would be awesome to have a field to create a confirmation that the user has to give in order for the submit to succeed.
This would be needed, for example, for acceptance of terms and conditions, privacy policies, and similar things.

### PR DESCRIPTION
**Pull request by nzambello**
Fixes #1 

**BREAKING CHANGE**

Renames `checkbox` field to `multiple_choice`
Renames `radio` field to `single_choice`
Adds `checkbox` as a single confirmation checkbox.

Screenshot 2021-08-07 at 18-23-34 Sito

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: src/components/Field.jsx [hunk 1]
```diff
@@ -84,8 +85,8 @@ const Field = ({
           name={name}
           title={label}
           description={description}
-          getVocabulary={() => {}}
-          getVocabularyTokenTitle={() => {}}
+          getVocabulary={() => { }}
+          getVocabularyTokenTitle={() => { }}
```
Comment (pnicolli) [inline src/components/Field.jsx:89]:
```suggestion
          getVocabulary={() => {}}
          getVocabularyTokenTitle={() => {}}
```

File: src/components/Sidebar.jsx [hunk 2]
```diff
@@ -398,8 +402,8 @@ const Sidebar = ({
                     {selected === index ? (
                       <Icon name={upSVG} size="20px" />
                     ) : (
-                      <Icon name={downSVG} size="20px" />
-                    )}
+                        <Icon name={downSVG} size="20px" />
+                      )}
```
Comment (pnicolli) [inline src/components/Sidebar.jsx:406]: Not sure if this should be indented like this.. was it checked by prettier? Just double checking to be 100% sure.

File: src/components/Sidebar.jsx [hunk 3]
```diff
@@ -484,22 +492,22 @@ const Sidebar = ({
...
+                          onChange={(name, value) => {
+                            onChangeSubBlock(index, {
+                              ...subblock,
+                              [name]: value,
+                            });
+                          }}
+                          required={true}
+                          value={subblock.input_values}
+                        />
+                      )}
```
Comment (pnicolli) [inline src/components/Sidebar.jsx:510]: Not sure if this should be indented like this.. was it checked by prettier? Just double checking to be 100% sure.

File: src/components/Sidebar.jsx (hunk 2 above)
Comment (nzambello) [reply src/components/Sidebar.jsx]: Yep, vim sometimes messes up with prettier 😞

**Comment by nzambello** [comment]
This may need an upgrade step in https://github.com/collective/collective.volto.formsupport to update existing block
@tiberiuichim @cekk I need your help here 😅

**Review by ericof — APPROVED** [review]

**Comment by ericof** [comment]
Before release, I will update the Brazilian translation

**Comment by cekk** [comment]
@nzambello @ericof please test this: https://github.com/collective/collective.volto.formsupport/pull/7

**Pull request merged**